### DISCUSSION TITLE
Error running atlantafx in IntelliJ: stack - java: module not found: atlantafx.base

### DISCUSSION THREAD

**Discussion by podwesley** [mkpaz/atlantafx#91] [General]
Hello everyone! I'm encountering an error while running atlantafx on IntelliJ. After adding the dependency in Maven and in the runner, the console shows the following error: java: module not found: atlantafx.base

Does anyone have any idea what this could be? See the image below:
![image](https://github.com/mkpaz/atlantafx/assets/29009797/b4cb656c-0408-44cd-b35d-0c11c4a32849)

**Comment by devrj8** [comment]
@podwesley 
Add the module to module-info.java file

![image](https://github.com/mkpaz/atlantafx/assets/25859158/857e9a46-51d3-4dbe-90b7-5f3082cf894f)
